package.json

```diff
@@ -66,6 +66,7 @@
   },
   "dependencies": {
     "eventemitter3": "^3.1.0",
+    "apache-arrow": "^0.17.0",
```

Comment:
v2.0.0 is out already.

Reply:
As with Thrift, we need to keep this pinned to the same version the backend is built with in order to guarantee format compatibility. I [do see that we've upgraded](https://github.com/omnisci/omniscidb/blob/master/scripts/common-functions.sh#L66) to arrow 1.0.0 so I will bump this accordingly and add a note.

Reply:
Scratch that, it looks like they never cut a release for 1.0.0 on NPM, and there are no breaking changes made to the JS connector from 1.0.0 to 2.0.0, so 2.0.0 it is 👍

Reply:
Note that the library version and format versions are not the same. Since 1.0, the format is Arrow 1.0. See https://arrow.apache.org/docs/format/Versioning.html